rules/no-identical-title.js

```diff
@@ -51,10 +64,10 @@ module.exports = {
         if (isDescribe(node)) {
           contexts.push(newDescribeContext());
         }
-        if (!isFirstArgLiteral(node)) {
+        const [firstArgument] = node.arguments;
```

Comment:
I think that `firstArgument` can technically be undefined. I'm not 100% sure we are protecting against that case. For example, the following code:

```js
describe()
```

produces the following AST

```json
{
  "type": "Program",
  "start": 0,
  "end": 10,
  "range": [
    0,
    10
  ],
  "body": [
    {
      "type": "ExpressionStatement",
      "start": 0,
      "end": 10,
      "range": [
        0,
        10
      ],
      "expression": {
        "type": "CallExpression",
        "start": 0,
        "end": 10,
        "range": [
          0,
          10
        ],
        "callee": {
          "type": "Identifier",
          "start": 0,
          "end": 8,
          "range": [
            0,
            8
          ],
          "name": "describe"
        },
        "arguments": []
      }
    }
  ],
  "sourceType": "module"
}
```

So `node.arguments[0] === undefined`.

Reply:
Good catch, should be a test case

Reply:
Yep totally agree, will add 👍